packages/styles/src/tile.scss

```diff
@@ -831,25 +850,50 @@ $fd-line-tile-badge-horizontal-spacing: 0.375rem !default;
       }
 
       .#{$block}__action-close {
-        @include fd-set-height(1.625rem);
-        @include reset-position-absolute();
+        @include fd-square(1.375rem);
         @include fd-flex-center();
+        @include fd-set-position-right(-0.425rem);
+
 
+        position: relative;
```

Comment:
How are these 2 things working? To set a position right (left, top, bottom), you need `position: absolute`, o.w. it's not working. This mixin `@include fd-set-position-right(-0.425rem);` should not be working with line 858 `position: relative;`